src/leap/common/http.py

```diff
@@ -76,17 +196,27 @@ def __init__(self, cert_file=None, pool=_pool):
         :param pool: An optional dedicated connection pool to override the
                      default shared one.
         :type pool: HTTPConnectionPool
+        :param timeout: The amount of time that this Agent will wait for the
```

Comment:
I think it should be the same to reduce complexity from the perspective of the user of HTTPClient.